src/web/app/pages-instructor/instructor-search-page/student-result-table/student-result-table.component.html

```diff
@@ -1,5 +1,4 @@
 <div class="card border border-primary mb-4">
-  <div class="card-header text-white bg-primary font-weight-bold">Students</div>
```

Comment:
It currently looks like this after the changes:
<img width="1440" alt="Screen Shot 2022-02-12 at 10 37 59 PM" src="https://user-images.githubusercontent.com/60355570/153715736-64cce049-2ebc-4891-8ec6-11bef45cbd08.png">

The blue outline is a little jarring and feels out of place. The header has been removed, but the enclosing div still remains. How about we removed the outermost div entirely? Something like this will look a little better:

<img width="1440" alt="Screen Shot 2022-02-12 at 10 37 05 PM" src="https://user-images.githubusercontent.com/60355570/153715980-a64a6511-b34a-48c5-8078-9eae3c1bfdec.png">